app/main/views/suppliers.py

```diff
@@ -50,3 +54,89 @@ def get_supplier(supplier_id):
     ).first_or_404()
 
     return jsonify(suppliers=supplier.serialize())
+
+
+# Route to insert new Suppliers, not update existing ones
+@main.route('/suppliers/<int:supplier_id>', methods=['PUT'])
```

Comment:
This endpoint will have to overwrite any existing record with the given ID, otherwise we won't be able to import data to the prod databases without deleting everything first.